instances/bodySystem/centralNervousSystem.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/bodySystem/centralNervousSystem",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySystem",
+  "definition": "In vertebrates 'central nervous system' consists of the neural tube derivatives: the brain and spinal cord. In invertebrates it includes central ganglia plus nerve cord.",
+  "description": "The central nervous system is the core nervous system that serves an integrating and coordinating function.",
```

Comment:
Better to flip them around:

definition: "The 'central nervous system' is the main processing center in most organisms. It's function is to take in sensory information, process information, and send out motor signals."

description: "In vertebrates, the central nervous system (CNS) consists of the brain and the spinal cord. In invertebrates, it includes the central ganglia and the nerve cord."